9.3_rotbinsearch.js

```diff
@@ -0,0 +1,101 @@
+
+//finds the position of the min value in rotated increasingly sorted array
+var binSearchMin = function (arr, min, max) {
+	if (max - min < 2){
+		return 0;
+	}
+	if (arr[min] < arr[max] || (min - 1 > 0 && arr[min -1] > arr[min])) {
+		return min;
+	} else { //arr[min] >= arr[max]
```

Comment:
Consider taking the code out of this else. The above if statement returns so function will exit at that point